### DISCUSSION TITLE
Welcome to www.javascript.fun Discussions!

### DISCUSSION THREAD

**Discussion by im6** [im6/javascript-fun#7] [General]
## 👋 Welcome to www.javascript.fun
Free feel to leave a comment of any suggestion and feedback, such as:
  * recommend some new javascript package or website?
  * have different idea on how we categorize certain package or library?
  * bug Reports.

**Comment by IOLOII** [comment]
# Nice website !

**Comment by UshakovMaksym** [comment]
In rating listed VUE 2, add please VUE 3: https://github.com/vuejs/core

**Reply by im6** [reply]
thanks for suggestion. I have already added vue 2 and 3 respectively
